### DISCUSSION TITLE
一款 Typora 主题

### DISCUSSION THREAD

**Discussion by giscus** [loveminimal/comment#27] [General]
# 一款 Typora 主题

一款 Typora 主题 - https://aituyaa.com/%E4%B8%80%E6%AC%BE-typora-%E4%B8%BB%E9%A2%98/

https://aituyaa.com/%E4%B8%80%E6%AC%BE-typora-%E4%B8%BB%E9%A2%98/

**Comment by Comrade-Ming** [comment]
就是可能正文的字体太小了，调成24 也好小，嘿嘿希望博主更新更新

**Reply by loveminimal** [reply]
之前使用多个分辨率设备都是正常的哟 ~  

如果你想修改的大一点的话，更改 `virgo.css` 的一片位置估计不行，因为它是编译后的文件。源文件是 `virgo.scss` 中引用的那些 `*.scss` 文件。

你可以修改 https://github.com/loveminimal/typora-theme-virgo/blob/5b243e5b2f6e89c061a38fe8e461531ca643632f/virgo/scss/custom.scss#L10 这一行，并且重新编译为 `virgo.scss`  和 `virgo-dark.scss` 为对应的 css 文件。
